nano/secure/ledger.cpp

```diff
@@ -939,10 +944,9 @@ bool nano::ledger::is_send (nano::transaction const & transaction_a, nano::state
 	}
 	else
 	{
-		nano::block_hash previous (block_a.hashables.previous);
 		if (!previous.is_zero ())
 		{
-			if (block_a.hashables.balance < balance (transaction_a, previous))
+			if (block_a.balance () < balance (transaction_a, previous))
```

Comment:
Does this hurt perf vs looking at the hashables? I think it shouldn't though.

Reply:
It doesn't show up in profiles. Basically until it does we should default to using encapsulation.